src/monitormanager.h

```diff
@@ -31,6 +32,7 @@ class MonitorManager : public IndexingObject<Monitor> {
     int list_monitors(Output output);
     int list_padding(Input input, Output output);
     int string_to_monitor_index(std::string string);
+    void completeMonitorName(Completion& complete);
```

Comment:
This looks and feels like an internal helper. Why not make it private?

Reply:
It should have the same visibility as `string_to_monitor_index` because every command that is allowed to take a monitor as an argument should be able to call `completeMonitorName`. (That's why the other completion helpers in other classes are (and should stay) public too, e.g. `RootCommands::completeAttributePath`)

Reply:
Okay, makes sense! :+1: